core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -2145,6 +2145,24 @@ class LogTest {
   }
 
   @Test
+  def testLogDeletionAfterClose() {
```

Comment:
Is this test related to this change? Not an issue if it's not, but just making sure I understand.

Reply:
This test is not related to this patch. This test is from https://github.com/apache/kafka/pull/4100/files which we back ported to 0.11 branch. In that pull request I said I will add this test to a future patch in the trunk.

Reply:
Can you please mention in the PR description?

Reply:
Sure. I added a sentence in the PR description to note that this patch adds a test to test log deletion after close.